apps/hash-graph/lib/graph/src/knowledge/entity/query.rs

```diff
@@ -355,12 +355,7 @@ impl<'de> Visitor<'de> for EntityQueryPathVisitor {
             EntityQueryToken::Type => EntityQueryPath::Type(
                 EntityTypeQueryPathVisitor::new(self.position).visit_seq(seq)?,
             ),
-            EntityQueryToken::Properties => {
-                let property = seq
-                    .next_element()?
-                    .ok_or_else(|| de::Error::invalid_length(self.position, &self))?;
```

Comment:
Do we not want to throw this error or is it just not needed anymore?

Reply:
We don't want to as we can compare the whole JSON object